modules/decision/decision_worker.py

```diff
@@ -0,0 +1,75 @@
+"""
+Retrieves list of landing pads from cluster estimation and outputs decision to the flight controller 
+
+"""
+
+from utilities.workers import queue_proxy_wrapper
+from utilities.workers import worker_controller
+from . import decision
+from . import landing_pad_tracking
+from . import search_pattern
+from ..flight_interface.flight_interface_worker import current_odometry, odometry_mutex
+
+
+def decision_worker(
+    camera_fov_forwards: float,
+    camera_fov_sideways: float,
+    search_height: float,
+    search_overlap: float,
+    distance_squared_threshold: float,
+    small_adjustment: float,
+    tolerance: float,
+    cluster_input_queue: queue_proxy_wrapper.QueueProxyWrapper,
+    output_queue: queue_proxy_wrapper.QueueProxyWrapper,
+    controller: worker_controller.WorkerController,
+) -> None:
+    """
+    Worker process.
+
+    PARAMETERS
+    ----------
+        - camera_fov_forwards and camera_fov_sideways are the measurements for the cameras field of view
+        - search_height and search overlap are the parameters for the search pattern
+        - distance_squared_threshold, small adjustment, and tolerance are the initial settings
```

Comment:
No need to repeat the descriptions, just write that they are for the constructors.